superset-frontend/src/components/Select/AntdSelect.tsx

```diff
@@ -0,0 +1,311 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, {
+  ReactElement,
+  ReactNode,
+  UIEvent,
+  useEffect,
+  useMemo,
+  useState,
+  useRef,
+} from 'react';
+import { styled } from '@superset-ui/core';
+import { Select as AntdSelect, Spin } from 'antd';
+import {
+  SelectProps as AntdSelectProps,
+  SelectValue as AntdSelectValue,
+  LabeledValue as AntdLabeledValue,
+} from 'antd/lib/select';
+import debounce from 'lodash/debounce';
+import { hasOption } from './utils';
+
+type AntdSelectAllProps = AntdSelectProps<AntdSelectValue>;
+type PickedSelectProps = Pick<
+  AntdSelectAllProps,
+  | 'allowClear'
+  | 'autoFocus'
+  | 'value'
+  | 'defaultValue'
+  | 'disabled'
+  | 'filterOption'
+  | 'loading'
+  | 'mode'
+  | 'notFoundContent'
+  | 'onChange'
+  | 'placeholder'
+  | 'showSearch'
+  | 'value'
+>;
+export type OptionsType = Exclude<AntdSelectAllProps['options'], undefined>;
+export type OptionsPromise = (
+  search: string,
+  page?: number,
+) => Promise<OptionsType>;
+export enum ESelectTypes {
+  MULTIPLE = 'multiple',
+  TAGS = 'tags',
+  SINGLE = '',
+}
+export interface SelectProps extends PickedSelectProps {
+  allowNewOptions?: boolean;
+  ariaLabel: string;
+  header?: ReactNode;
+  name?: string; // discourage usage
```

Comment:
I'm curious why this a prop we don't want to use anymore?

Reply:
There's has been a discussion for this prop. The consensus is that `aria-label `is a better option